File: src/main/java/help/commands/store/UserCommands.java
package help.commands.store;

import help.commands.dto.commands.HelpCommandType;
import help.commands.dto.commands.HelpCommands;
import help.commands.dto.response.CommandsResponse;
import help.commands.dto.response.CommandsResponseItem;
import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Component;

import java.util.List;

@Component
public class UserCommands {

    public ResponseEntity<CommandsResponse> userCommands(){
        return ResponseEntity.ok().body(
                new CommandsResponse(
                    List.of(
                        new CommandsResponseItem(
                            HelpCommandType.REGISTRATION,
                            new HelpCommands(
                                   "This link will allow you to register",
                                    "/auth/registration"
                            )
                    ),
                            new CommandsResponseItem(
                                    HelpCommandType.REFRESH_TOKEN,
                                    new HelpCommands(
                                            "This link will allow you to refresh your JWT",
                                            "/auth//token/refresh"
                                    )
                            ),
                            new CommandsResponseItem(
                                    HelpCommandType.WRITE,
                                    new HelpCommands(
                                            "This link will allow you to write an appeal",
                                            "/appeal/write"
                                    )
                            )
                    )

                )
        );
    }
}

File: src/main/java/help/commands/store/UserCommands.kt
package help.commands.store

import help.commands.dto.commands.HelpCommandType
import org.springframework.stereotype.Component

@Component
class UserCommands {
    private val commands: Map<HelpCommandType, HelpCommands> = mapOf(
        HelpCommandType.REGISTRATION to HelpCommands(
            description = "This command will allow you to register yourself, or register your new account",
            command = "user -register",
        ),
        HelpCommandType.CHOOSE_CITY to HelpCommands(
            description = "This command will allow you to select a city if your registered in the application",
            command = "user -choose --city"
        ),
        HelpCommandType.WRITE_APPEAL to HelpCommands(
            description = "This command will allow you to write an appeal if you have selected your city",
            command = "user -write --appeal"
        ),
        HelpCommandType.CHANGE_INFO to HelpCommands(
            description = "This command will allow you to change information in your profile",
            command = "user -change --info"
        )
    )

    fun getAll() = commands
}